src/main/java/org/opensearch/knn/index/IndexUtil.java

```diff
@@ -139,17 +139,16 @@ public static ValidationException validateKnnField(
 
             ModelMetadata modelMetadata = modelDao.getMetadata(modelId);
             if (modelMetadata == null) {
-                exception.addValidationError(String.format("Model \"%s\" for field \"%s\" does not exist.", modelId, field));
```

Comment:
I think these messages are useful for customer to identify issues with a particular model, especially when they have multiple models.